smarts/core/coordinates.py

```diff
@@ -258,12 +256,20 @@ class Pose:
     """A pair of position and orientation values."""
 
     # TODO: these should be np.ndarray
-    position: Union[NamedTuple, Tuple, np.ndarray]  # [x, y, z]
-    orientation: Union[
-        NamedTuple, Tuple, np.ndarray
-    ]  # [a, b, c, d] -> a + bi + cj + dk = 0
+    position: np.ndarray  # [x, y, z]
+    orientation: np.ndarray  # [a, b, c, d] -> a + bi + cj + dk = 0
     heading_: Optional[Heading] = None  # cached heading to avoid recomputing
 
+    def __post_init__(self):
+        if not isinstance(self.position, np.ndarray):
+            self.position = np.array(self.position, dtype=np.float64)
```

Comment:
You should use `dtype=float` since `np.float64` is deprecated in version of numpy.

Reply:
`np.float64` does not seem to be deprecated although `np.float` is.